MC/bin/o2dpg_sim_config.py

```diff
@@ -53,21 +53,52 @@ def add(cfg, flatconfig):
-          add(config, {"pvertexer.dbscanDeltaT" : 7,
+          # remove extra errors on time margin for tracks and ITS clusters
+          add(config, {"pvertexer.timeMarginTrackTime" : 0.,
+                       "pvertexer.dbscanDeltaT" : 7,
                        "pvertexer.maxChi2TZDebris": 50,
                        "pvertexer.maxMultRatDebris": 1.,
                        "pvertexer.dbscanAdaptCoef" : 20,
                        "pvertexer.maxVerticesPerCluster" : 20,
-                       "pvertexer.dbscanMaxDist2" : 36})
-
+                       "pvertexer.dbscanMaxDist2" : 36,
+                       "ITSCATrackerParam.sysErrY2[0]": 0.,
+                       "ITSCATrackerParam.sysErrY2[1]": 0.,
+                       "ITSCATrackerParam.sysErrY2[2]": 0.,
+                       "ITSCATrackerParam.sysErrY2[3]": 0.,
+                       "ITSCATrackerParam.sysErrY2[4]": 0.,
+                       "ITSCATrackerParam.sysErrY2[5]": 0.,
+                       "ITSCATrackerParam.sysErrY2[6]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[0]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[1]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[2]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[3]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[4]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[5]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[6]": 0.})
        else:
           # generic pp
-          add(config, {"pvertexer.acceptableScale2" : 9,
+          # remove extra errors on time margin for tracks and ITS clusters
+          add(config, {"pvertexer.timeMarginTrackTime" : 0.,
+                       "pvertexer.acceptableScale2" : 9,
                        "pvertexer.dbscanMaxDist2" : 36,
                        "pvertexer.dbscanDeltaT" : 24,
                        "pvertexer.maxChi2TZDebris" : 100,
                        "pvertexer.maxMultRatDebris" : 1.,
-                       "pvertexer.dbscanAdaptCoef" : 20.})
+                       "pvertexer.dbscanAdaptCoef" : 20.,
+                       "ITSCATrackerParam.sysErrY2[0]": 0.,
+                       "ITSCATrackerParam.sysErrY2[1]": 0.,
+                       "ITSCATrackerParam.sysErrY2[2]": 0.,
+                       "ITSCATrackerParam.sysErrY2[3]": 0.,
+                       "ITSCATrackerParam.sysErrY2[4]": 0.,
+                       "ITSCATrackerParam.sysErrY2[5]": 0.,
+                       "ITSCATrackerParam.sysErrY2[6]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[0]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[1]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[2]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[3]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[4]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[5]": 0.,
+                       "ITSCATrackerParam.sysErrZ2[6]": 0.})
```

Comment:
@fgrosa note that this will also affect the ITS tracking and matching to TPC (less fakes will be produced).
In general, we try to use for anchored MC the same settings as used for data. Is it worth changing these settings globally? Perhaps one should do first a dedicated production to test its effect? 
Relaxed timing tolerances lead to false-positive `ambiguous` tracks, which "only" increases the combinatorial background but should not lead to inefficiency (once ambiguities are accounted in the analysis).
Also, note this fix: https://alice.its.cern.ch/jira/browse/O2-3054?focusedCommentId=289529&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-289529, thought it was affecting mostly MFT, for the ITS it effectively added <1\mus extra tolerance.

Reply:
Hi @shahor02 thanks a lot! I removed the settings in the case of `302000 <= run <= 309999` which is the one for the high energy pp program. In general, I agree with you, however HF and DQ vertexing codes are not yet designed to account for ambiguous tracks and it was suggested by @mpuccio and @ginnocen to look at the inefficiency that we obtain removing this extra tolerances.